packages/renderer/src/lib/actions/ContributionActions.svelte

```diff
@@ -98,12 +99,13 @@ async function executeContribution(menu: Menu): Promise<void> {
 </script>
 
 {#each filteredContributions as menu}
-  <ListItemButtonIcon
-    title="{menu.title}"
-    onClick="{() => executeContribution(menu)}"
-    menu="{dropdownMenu}"
-    icon="{getIcon(menu)}"
-    detailed="{detailed}"
-    disabledWhen="{menu.disabled}"
-    contextUI="{globalContext}" />
+  <When expression="{menu.disabled}" contextUI="{globalContext}" let:result>
+    <ListItemButtonIcon
```

Comment:
I have the feeling that we should not use a UI component for setting a value

I would expect
```
{@const isEnabled = ..... }
<ListItemButtonIcon
  enabled={isEnabled}
```

Reply:
https://github.com/containers/podman-desktop/pull/7658#issuecomment-2188297551